homeassistant/components/local_ip/sensor.py

```diff
@@ -24,11 +24,21 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
```

Comment:
Should remove this as well per the unique id requirements https://developers.home-assistant.io/docs/entity_registry_index/

Reply:
The goal of this PR was to make the sensor visible from the integration panel.

Just adding the `unique_id` solves this, so it's not possible to remove it (or I don't nkow the solution to make it work).

Perhaps we can use the name or the domain.

The domain should be used because of `single_instance_allowed`, but aleady prepend, as written in the doc.

Also notices that with the unique_id, the sensor is registred with the "Locale IP" integration, without it's registered on the sensor domain.

Reply:
@MartinHjelmare Is this ok since its single instance only?

Reply:
https://github.com/home-assistant/core/pull/33483#discussion_r401973309